include/partition/graph_view.hpp

```diff
@@ -79,6 +79,9 @@ class GraphView
     EdgeIterator EdgeBegin(const NodeID nid) const;
     EdgeIterator EdgeEnd(const NodeID nid) const;
 
+    const BisectionNode &GetNode(const NodeID nid) const;
+    const BisectionEdge &GetEdge(const EdgeID eid) const;
```

Comment:
I started adapting the graph we're holding on to in the graph view - we probably need more functionality here anyway, for the flow algorithm to operate on.